VolleyPaint/Assets/Scripts/Ability/AbilityHolder.cs

```diff
@@ -2,9 +2,11 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Events;
+using Unity.Netcode;
 
-public class AbilityHolder : MonoBehaviour
+public class AbilityHolder : NetworkBehaviour
```

Comment:
these seem like Kims changes so we should probably merge all his changes from the ability test branch first

Reply:
I think those are Kim's, yeah.

Reply:
I think we should merge Kim's branch when he's ready. Keeping this change for now shouldn't cause a merge conflict, right?